package/kedro_viz/server.py

```diff
@@ -317,90 +330,143 @@ def _construct_layer_mapping(catalog):
     return dataset_to_layer
 
 
-# pylint: disable=too-many-locals
-def format_pipeline_data(pipeline, catalog):
+def _pretty_name(name: str) -> str:
+    name = name.replace("-", " ").replace("_", " ")
+    parts = [n[0].upper() + n[1:] for n in name.split()]
+    return " ".join(parts)
+
+
+def format_pipelines_data(pipelines: Dict[str, "Pipeline"], catalog) -> Dict[str, list]:
     """
-    Format pipeline and catalog data from Kedro for kedro-viz
+    Format pipelines and catalog data from Kedro for kedro-viz.
 
     Args:
-        pipeline: Kedro pipeline object
-        catalog:  Kedro catalog object
-    """
+        pipelines: Dictionary of Kedro pipeline objects.
+        catalog: Kedro catalog object.
 
-    def pretty_name(name):
-        name = name.replace("-", " ").replace("_", " ")
-        parts = [n[0].upper() + n[1:] for n in name.split()]
-        return " ".join(parts)
+    Returns:
+        Dictionary of pipelines, nodes, edges, tags and layers, and pipelines list.
 
+    """
+    pipeline_list = []
```

Comment:
```suggestion
    pipelines_list = []
```

The same suggestion for `node_` and `edge_`.